packages/shared/src/utils/contract/contract-types.ts

```diff
@@ -15,30 +15,55 @@ export type TIsEnded = Partial<TGetFinalPrice> & {
 };
 
 export type TContractInfo = {
```

Comment:
@maryia-deriv Just a question, Don't we have a type for `Contract` in the `@deriv/api-types` or this is not a API response? 🤔

Reply:
@farzin-deriv yes, i've extended TContractInfo with a type from @deriv/api-types for the purposes of TS migration of turbos-card-body. However, its full replacement needs to be done outside of the current PR as TContractInfo was introduced earlier, replacement of all its occurances is required in shared package, and we already have 100 files in our PR. thank you for pointing this out too!🙂